desktop-exporter/app/components/traceList.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useRef } from "react";
+import { FixedSizeList } from "react-window";
+import { NavLink } from "react-router-dom";
+import { Flex } from "@chakra-ui/react";
+import { useSize } from "@chakra-ui/react-use-size";
+
+import { TraceSummary } from "../types/api-types";
+
+type RowProps = {
+  index: number;
+  style: Object;
+  data: TraceSummary[];
+};
+
+function Row({ index, style, data }: RowProps) {
+  return (
+    <NavLink
+      to={`traces/${data[index].traceID}`}
+      style={style}
+    >
+      {data[index].traceID}
+    </NavLink>
+  );
+}
+
+type TraceListProps = {
+  traceSummaries: TraceSummary[];
+};
+
+export function TraceList(props: TraceListProps) {
+  const ref = useRef(null);
+  const size = useSize(ref);
+
+  return (
+    <Flex
+      ref={ref}
+      height="100%"
+    >
+      <FixedSizeList
+        className="list"
```

Comment:
Do we still need this `className`? There are no rules for it left

Reply:
We do not